gdpr-samples/gdpr-qsg/src/main/resources/gdpr.sh

```diff
@@ -764,34 +777,92 @@ echo "** Service Provider ${sp_name} successfully deleted. **"
 return 0;
 }
 
-echo Please pick a scenario from the following.
+echo Please start from step 1:.
 echo "----------------------------------------------------------------"
+echo "| This is the Quick Start Guide for GDPR demonstrations.        |"
+echo "| =====================================================         |"
+echo "| Before run this make sure your WSO2 IS and Tomcat is running  |"
+echo "| in default ports WSO2 IS - localhost:9443                     |"
+echo "|                  tomcat  - localhost:8080                     |"
+echo "| Next, Try the below steps in order                            |"
```

Comment:
Shall we have above part edited as below?
```
echo "| Before running this, make sure your WSO2 IS and Tomcat is     |"
echo "| running in default ports.                                     |"
echo "|        WSO2 IS - localhost:9443                               |"
echo "|        tomcat  - localhost:8080                               |"
echo "| Next, try below steps in order.                               |"
```

Reply:
This is releasing basically for IS 5.7.0, due to API changes in some admin services of IS 5.7.0, this QSG does not work for lower versions.